docs/source/gbd2017_models/causes/covid_19/index.rst

```diff
@@ -11,35 +11,285 @@ Covid-19 Custom SEIR Vivarium Cause Model
+
+   Add a general clinical overview of the covid-19, any relevant research.
+
+:math:`R_{0}` will be defined here as the population level average number of infections a single individual with covid-19 will transmit over the entire infectious period assuming a completely susceptible population and *typical* mobility and population density.
+
+  Notably, :math:`R_0` is approximately equal to the average number of social contacts per person per day multiplied by the transmission probability of a contact between an infectious individual multiplied by the average duration of the infectious period.
+
+:math:`R_{0}'` will be defined here as the population level average number of infectiouns a single individual with covid-19 will transmit over the entire infectious period assuming a completely susceptible population and a *given level* of mobility and population density.
+
+:math:`R_{t}` (or effective :math:`R`) will be defined here as the population average number of infections a single individual with covid-19 will transmit over the entire infectious period at a given point in time.
+
+Notably, while :math:`R_{0}`, :math:`R_{0}'`, and :math:`R_{t}` are population level average parameters, the distribution of these parameters at the *individual level* will be highly right skewed, indicating that many infectious individuals will transmit fewer infections than the population mean R value, while some individuals will transmit many more infections that the population mean value (e.g. mass transmission events).
+
 IHME Covid Team Modeling Strategy
 ---------------------------------
 
+Covid-19 is not included in currently available GBD study data. Rather, we will use custom data for all parameters in this cause model using available data from the existing IHME covid-19 forecasting model.
+
+IHME's existing covid-19 models have fit covariate estimates for **population density,  relative mobility, testing capacity, and temperature.** values over time. Below is a brief overview of the SEIR component of the IHME forecast model.
+
+The steps that the SEIR component of the forecast model follows are:
+
+#. Fit an SEIR model (e.g. fit :math:`\beta(t)`) to past and recent death model output for all locations
+
+#. Regress :math:`\beta(t)` on available covariates
+
+#. Forecast time-varying covariates into the future
+
+#. Combine regression with forecasts to forecast :math:`\beta(t)`
+
+#. Run forecasted :math:`\beta(t)` through SEIR model to forecast infections
+
+#. Calculate deaths from infections and infection fatality ratio (IFR)
+
+Notably, the IHME forecast SEIR component includes a dual infectious state, :math:`I_{1}` and :math:`I_{2}`. The following equations represent the differential equations for each state in the SEIR component used in the IHME forecast model:
+
+			:math:`S` --> :math:`E` --> :math:`I_1` --> :math:`I_2` --> :math:`R`
+
+:math:`dS/dt = -(\beta(t)S(I_1+I_2)^\alpha)/N`
+
+:math:`dE/dt = (\beta(t)S(I_1+I_2)^\alpha)/N - \sigma E`
```

Comment:
Let A and B be the two groups of susceptible people that infectious people, person_a and person_b come into contact with. Do these assume that A and B are disjoint?

Reply:
I'm not sure I understand the question. This wasn't in the version that you reviewed, but essentially I1 = pre-symptomatic and I2 = symptomatic, although both I1 and I2 have infectious potential. Does that help clarify?

Reply:
My question is essentially:

I understand beta(t) to be an estimate of how many people the average infected person infects. I am then interpreting the people _entering_ E to be calculated by multiplying beta by the # of infectious individuals

That seems to assume that infectious people A and B don't both contribute to getting susceptible person C sick ---that people A and B each get their own, discrete sets of people sick

But it sounds like you're accounting for this perhaps with alpha? (How does alpha work/how is it chosen?)